homework7/7.5_task.py

```diff
@@ -0,0 +1,12 @@
+def my_all(list) -> bool:
```

Comment:
shadow build_in naming. Try to avoid names such list,set,tuple .... in your functions